x-pack/test/api_integration/apis/management/rollup/index_patterns_extensions.js

```diff
@@ -65,9 +65,9 @@ export default function ({ getService }) {
         const { body } = await supertest.get(uri).expect(200);
 
         // Verify that the fields for wildcard correspond to our declared mappings
-        const propertiesWithMappings = Object.keys(INDEX_TO_ROLLUP_MAPPINGS.properties);
+        // noting that testTotalField and testTagField are not shown in the field caps results
         const fieldsForWildcard = body.fields.map((field) => field.name);
-        expect(fieldsForWildcard.sort()).eql(propertiesWithMappings.sort());
+        expect(fieldsForWildcard.sort()).eql(['testCreatedField']);
```

Comment:
The existing test relied on the broken behavior - returning fields that didn't have proper field caps entries. Simply adjusted the test to expect the field with complete info.